LDM-148.tex

```diff
@@ -636,8 +643,10 @@ \section{Offline Production Enclave}\label{offline-production-enclave}
 the DMS: the annual Data Release Production (DRP) and periodic
 Calibration Products Productions (CPPs). Note that CPPs will execute
 even while the annual DRP is executing.
- The Offline Quality Control Service monitors the science data
+The Offline Quality Control Service monitors the science data
 products, notifying operators if any anomalies are found.
+The enclave also includes a service for distributing bulk data on daily and annual (Data Release) timescales to partner institutions, collaborations, and LSST Education and Public Outreach (EPO).
+This bulk data transfer may occur in large batches (e.g. once per day or when a Data Release is about to occur) or in narrower streams (e.g. as candidate data products become available from the Data Release Production).
```

Comment:
This appears to confirm the move of the Bulk Distribution service that I queried above. Fair enough, but then fig 3. really does need an update.